include/RAJA/util/View.hpp

```diff
@@ -159,18 +159,20 @@ struct MultiView
   using NonConstView =
       MultiView<nc_value_type, layout_type, P2Pidx, nc_pointer_type>;
 
-  layout_type const layout;
-  nc_pointer_type data;
+  layout_type layout{};
+  nc_pointer_type data = nullptr;
+
+  MultiView() = default;
 
   template<typename... Args>
   RAJA_INLINE constexpr MultiView(pointer_type data_ptr, Args... dim_sizes)
       : layout(dim_sizes...),
-        data(data_ptr)
+        data(nc_pointer_type(data_ptr))
   {}
 
   RAJA_INLINE constexpr MultiView(pointer_type data_ptr, layout_type&& layout)
```

Comment:
I don't think layout needs to be taken by rvalue reference.
```suggestion
  RAJA_INLINE constexpr MultiView(pointer_type data_ptr, layout_type const& layout)
```